### ISSUE TITLE
🐛 Wrong biome build used for musl guest devcontainer on arm64 host

### ISSUE THREAD

**Issue by Mardoxx** [biomejs/biome-vscode#223]
### VS Code version

1.89.1

### Extension version

2.2.2

### Biome version

1.7.3

### Operating system

- [ ] Windows
- [X] macOS
- [ ] Linux

### Description

When using a musl based linux dev container on  a MacOS host (arm64) the wrong biome binary is used.

### Steps to reproduce

Init a new project and install biome., then set up dev container.

`.devcontainer/devcontainer.json`
```
{
  "name": "devcontainer",
  "build": {
    "dockerfile": "Dockerfile"
  },
  "postCreateCommand": "yarn install",
  "customizations": {
    "vscode": {
      "extensions": [
        "biomejs.biome"
      ]
    }
  }
}
```

`.devcontainer/Dockerfile`
```
FROM alpine:latest

RUN apk update && \
    apk add --no-cache git yarn
```

Biome finds `cli-linux-arm64`

### Expected behavior

Biome should find `cli-linux-arm64-musl`

`"biome.lspBin": "./node_modules/@biomejs/cli-linux-arm64-musl/biome"` makes it work

### Does this issue occur when using the CLI directly?

No

### Logs

```plain text
Biome binary found at /workspaces/demo/node_modules/@biomejs/cli-linux-arm64/biome
Copying binary to temporary folder: file:///root/.vscode-server/data/XXXXX/workspaceStorage/XXXXX/biomejs.biome/biome
Executing Biome from: /root/.vscode-server/data/XXXXX/workspaceStorage/XXXXX/biomejs.biome/biome
[Error - 5:27:47 PM] Biome client: couldn't create connection to server.
Error: spawn /root/.vscode-server/data/XXXXX/workspaceStorage/XXXXX/biomejs.biome/biome ENOENT
    at Process.ChildProcess._handle.onexit (node:internal/child_process:284:19)
    at onErrorNT (node:internal/child_process:477:16)
    at processTicksAndRejections (node:internal/process/task_queues:82:21)
[cli-stdout] end
[cli-stdout] finish
[cli-stderr] end
[cli-stderr] finish
[cli-stderr] close
[cli] close -2
[cli-stdout] close
```
```

**Comment by Mardoxx** [comment]
When installing biome in the guest, it also installs cli-linux-arm64 although correctly detecting musl base.

```
info Direct dependencies
└─ @biomejs/biome@1.7.3
info All dependencies
├─ @biomejs/biome@1.7.3
├─ @biomejs/cli-linux-arm64-musl@1.7.3
└─ @biomejs/cli-linux-arm64@1.7.3
```

**Comment by Mardoxx** [comment]
Maybe this needs tweaking:

https://github.com/biomejs/biome-vscode/blob/d9ca2b795f7f4faa81cc92806115e6bfec490e9b/src/main.ts#L465C1-L469C14

**Comment by nhedger** [comment]
Thanks, I'll take a look.

**Comment by nhedger** [comment]
Should be fixed in v2.2.3

**Comment by Mardoxx** [comment]
Fantastic thanks, seems to work
